packages/design-picker/src/utils/available-designs.ts

```diff
@@ -104,5 +104,24 @@ export function getAvailableDesigns( {
 		}
 	}
 
+	// Force blank canvas design to always be first in the list
+	const blankCanvasIndex = designs.featured.findIndex( ( design ) =>
+		isBlankCanvasDesign( design )
+	);
+	if ( blankCanvasIndex > -1 ) {
+		designs = {
+			...designs,
+			featured: [
+				designs.featured[ blankCanvasIndex ],
+				...designs.featured.slice( 0, blankCanvasIndex ),
+				...designs.featured.slice( blankCanvasIndex + 1 ),
+			],
+		};
+	}
+
 	return designs;
 }
+
+export function isBlankCanvasDesign( design: Design ): boolean {
+	return /blank-canvas/i.test( design.slug );
```

Comment:
Just a side note, practically speaking, are we going to have more than one black canvas to need this? I'd personally lock on one slug.

Reply:
There's going to be _at least_ 2 designs, one with slug `blank-canvas` and the other with slug `blank-canvas-blocks` (see #52295)